pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -2353,6 +2318,16 @@ func (dm *DockerManager) GetNetNS(containerID kubecontainer.ContainerID) (string
 	return netnsPath, nil
 }
 
+func (dm *DockerManager) GetPodContainerID(pod *kubecontainer.Pod) (*kubecontainer.ContainerID, error) {
+	for _, c := range pod.Containers {
+		if c.Name == PodInfraContainerName {
+			return &c.ID, nil
+		}
+	}
+
+	return nil, fmt.Errorf("Pod %v/%v unknown to docker.", pod.Namespace, pod.Name)
```

Comment:
@dcbw Please use `kubecontainer.BuildPodFullName(pod)` to make it consistent in multiple places.